tensortrade/exchanges/simulated/fbm_exchange.py

```diff
@@ -39,12 +39,23 @@ def __init__(self, **kwargs):
         self._start_date = self.default('start_date', '2010-01-01', kwargs)
         self._start_date_format = self.default('start_date_format', '%Y-%m-%d', kwargs)
         self._times_to_generate = self.default('times_to_generate', 100000, kwargs)
+        self._init_times_to_generate = self._times_to_generate
+        self._maintain_data_frame_len = self.default('maintain_data_frame_len', False, kwargs)        
         self._hurst = self.default('hurst', 0.61, kwargs)
         self._timeframe = self.default('timeframe', '1h', kwargs)
 
         self._generate_price_history()
 
     def _generate_price_history(self):
+        if self._maintain_data_frame_len:
+            if 'min' in self._timeframe:
+                self._times_to_generate = self._init_times_to_generate * int(self._timeframe[0])
```

Comment:
This would not work in the case of `10min`, as `self._timeframe[0]` would be `1`.

You would need to do something like `[int(s) for s in self._timeframe.split() if s.isdigit()]`